src/DtoTransformer/DtoTransformer/DtoTransformer.csproj

```diff
@@ -30,4 +30,10 @@
     <PackageReference Include="IriTools" Version="1.0.0" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Update="review.ttl">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+
```

Comment:
```suggestion
  <ItemGroup>
	  <EmbeddedResource Include="..\..\schema\review.ttl">
	    <Link>review.ttl</Link>
	    <IncludeInPackage>true</IncludeInPackage>
	    <CopyToOutputDirectory>Always</CopyToOutputDirectory>
	    <PackageCopyToOutput>true</PackageCopyToOutput>
	  </EmbeddedResource>
	</ItemGroup>

```

Reply:
Jeg tror dette lager ei lenke i stedet for å kopiere. Tatt fra records-biblioteket